### PR TITLE
Open Source FirebaseDynamicLinks

### PR DESCRIPTION
**Pull request by paulb777**
Open source DynamicLinks
Add FirebaseAnalyticsInterop
style
Add temporary FirebaseAnalytics podspecs until Firebase 5.9.0 is published.

I'll wait for travis green to push, but reviews accepted earlier.

Once this PR lands, I plan to publish FirebaseAnalyticsInterop to enable 'pod lib lint' testing for FirebaseDynamicLinks.

### CODE REVIEW

File: Example/DynamicLinks/App/iOS/FIRAppDelegate.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2017 Google
```
Comment (ryanwilson) [inline Example/DynamicLinks/App/iOS/FIRAppDelegate.h]: Looks like the example has 2017 as the copyright date - this should probably be updated to 2018.

File: FirebaseAnalyticsInterop.podspec [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseAnalyticsInterop'
+  s.version          = '1.0.0'
+  s.summary          = 'Interfaces that allow other Firebase SDKs to use Auth functionality.'
```
Comment (ryanwilson) [inline FirebaseAnalyticsInterop.podspec]: s/Auth/Analytics/

File: GoogleAppMeasurement.podspec.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (ryanwilson) [inline GoogleAppMeasurement.podspec.json]: This file should be removed.

File: Example/DynamicLinks/App/iOS/FIRAppDelegate.h (hunk 1 above)
Comment (paulb777) [reply Example/DynamicLinks/App/iOS/FIRAppDelegate.h]: I copied the harness app from an existing one, so kept the copyright year.

File: FirebaseAnalyticsInterop.podspec (hunk 2 above)
Comment (paulb777) [reply FirebaseAnalyticsInterop.podspec]: Good catch! Thanks.

File: GoogleAppMeasurement.podspec.json (hunk 3 above)
Comment (paulb777) [reply GoogleAppMeasurement.podspec.json]: This is needed for travis until 5.9.0 goes public.

Comment (ryanwilson) [reply GoogleAppMeasurement.podspec.json]: SGTM, my bad!

File: Example/DynamicLinks/App/iOS/FIRAppDelegate.h (hunk 1 above)
Comment (ryanwilson) [reply Example/DynamicLinks/App/iOS/FIRAppDelegate.h]: SGTM

**Pull request merged**